bigquery_etl/query_scheduling/dag_collection.py

```diff
@@ -110,8 +110,16 @@ def dag_to_airflow(self, output_dir, dag):
 
     def to_airflow_dags(self, output_dir, dag_to_generate=None):
         """Write DAG representation as Airflow dags to file."""
+        # https://pythonspeed.com/articles/python-multiprocessing/
+        # when running tests on CI that call this function, we need
+        # to create a custom pool to prevent processes from getting stuck
+        try:
+            set_start_method("spawn")
+        except Exception:
+            pass
```

Comment:
This wasn't necessary when running the tests locally. But not entirely sure what circleci changes, adding this became necessary.